packages/attention/src/component.tsx

```diff
@@ -1,154 +1,237 @@
-import React, { forwardRef, useEffect, useRef, useState } from "react";
-import { classNames } from "@chbphone55/classnames";
+import React, { forwardRef, useEffect, useRef, useState } from 'react'
+import { classNames } from '@chbphone55/classnames'
 import {
-  opposites,
-  rotation,
-  useRecompute as recompute,
-  arrowLabels,
-} from "@warp-ds/core/attention";
-import { attention as ccAttention } from "@warp-ds/css/component-classes";
-import { ArrowProps, AttentionProps } from "./props";
+	opposites,
+	rotation,
+	useRecompute as recompute,
+} from '@warp-ds/core/attention'
```

Comment:
Did we have some setting where the indent was supposed to be so big?

Reply:
No we need to fix a prettier-config, right now it's just the way my prettier formats the document 🙈

Reply:
Could you maybe change that to use 2 spaces instead of tabs? I think that was a configuration we had elsewhere at some point 😅

Reply:
Sure! :)